inc/based_config.php

```diff
@@ -46,9 +46,14 @@
    date_default_timezone_set(@date_default_timezone_get());
 }
 
+// If this file exists, it is load
+if (file_exists(GLPI_ROOT. '/config/local_define.php')) {
+   require_once GLPI_ROOT. '/config/local_define.php';
```

Comment:
cannot use GLPI_CONFIG_DIR which is not yet defined.
so probably need to be patched for downstream redirection